redaxo/src/core/lib/packages/addons/interface.php

```diff
@@ -30,28 +30,32 @@ public function pluginExists($plugin);
     /**
      * Returns the registered plugins.
      *
-     * @return rex_plugin[]
+     * @return array
```

Comment:
diese änderung ist blöd für phpstorm

Reply:
D.h. Besser das vorherige lassen, auch wenn es nicht korrekt ist?

Reply:
Warum ist es nicht korrekt?
`rex_plugin[]` ist `array<array-key, rex_plugin>`.
Das ist kein falscher Typ dafür, und er ist zumindest schon genauer als `array`.
Noch genauer ist halt, wenn man den Key aus `string` beschränkt, wie in der `@psalm-return`-zeile drunter.

Reply:
Es erzeugt zumindest eine warning.. diese ist aber durch den psalm-return auch gefixed

![7C22BD2D-BA48-4FDE-9226-31932B2B3BFC](https://user-images.githubusercontent.com/120441/76164053-ea537180-614b-11ea-921c-2e41d23b611a.png)

Reply:
Ja, der Typ war für psalm zu allgemein, trotzdem war er immerhin näher am eigentlichen typ, als `array`.
Und er stimmte trotzdem auch (so wie `array` auch stimmen würde), war bloß nicht genau genug angegeben.